_sass/layout.scss

```diff
@@ -94,6 +94,21 @@ header {
 	position: relative;
 }
 
+.video_wrapper {
+	position: relative;
+	padding-bottom: 56.25%;
```

Comment:
Does this work when resizing the screen and so on? Why did you choose a relative amount?

Reply:
I followed [this guide](https://learn.cloudcannon.com/jekyll/youtube/), in which it says "This forces the video to always be the full width of is container and to maintain a particular aspect ratio. In this case we have a ratio of 16:9 which gives us padding-bottom: 56.25%; (9/16)".

Reply:
This is how it would look like on a phone: 
![image](https://user-images.githubusercontent.com/41991517/99978362-2431da00-2da6-11eb-9f5b-24ae62857c5b.png)
And this for a medium sized display: 
![image](https://user-images.githubusercontent.com/41991517/99978444-3dd32180-2da6-11eb-9cba-c3d77b2befc2.png)

So it seems to work pretty well.